backend/catalogue/models.py

```diff
@@ -138,3 +138,12 @@ class BaseLayerGroup(models.Model):
 
     def __str__(self):
         return self.name
+
+
+@python_2_unicode_compatible
+class SerializedHashstate(models.Model):
+    id = models.CharField(max_length = 36, primary_key=True)
```

Comment:
Django does actually have a UUID field from memory (can't offhand remember if sqlserver has native support though...and obviously only useful if we use uuids anyway)